src/UICrafter.Client/Components/ComponentProperties.razor

```diff
@@ -42,6 +43,12 @@
             if (SelectedComponentChanged.HasDelegate && value != _internalSelectedComponent)
             {
                 _internalSelectedComponent = value;
+
+                if(_internalSelectedComponent?.ComponentCase == UIComponent.ComponentOneofCase.InputField)
+                {
+                    _internalSelectedComponent.InputField.LogicalName = _internalSelectedComponent.InputField.Label.ToLower().Replace(" ", "-");
```

Comment:
This looks good, you should however consider the amount of string manipulations you "string" together as each method allocates a new string as it is immutable. If a lot of modifications are needed a string builder makes more sense. There also exsist a lot of string optimizations in the framework that saves you most of the time. Its just good to keep in mind. 😄